packages/solidity/src/msgs.rs

```diff
@@ -167,8 +167,8 @@ impl TryFrom<ibc_core_client_types::Height> for IICS02ClientMsgs::Height {
 
     fn try_from(height: ibc_core_client_types::Height) -> Result<Self, Self::Error> {
```

Comment:
You should change this to `From` instead of `TryFrom`

Reply:
fixed